environments/autostart.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Autostart"
+description: Learn how to start your environments automatically at a predetermined time.
+---
+
+Autostart allows you to set environments to turn on automatically at a
```

Comment:
This seems redundant with the following section under background:

```
Autostart also allows you to configure automated environment rebuilds at a time
that best suits your workflow. You can now expect your environments to be ready
for you at the start of each workday.
```

Either delete that section under background or delete this imo.

I'm leaning more towards deleting this and just starting with the `Background` header.